src/main/resources/templates/account/passwordUpdated.html

```diff
@@ -0,0 +1,17 @@
+<page th:replace="fragments/user-layout :: page(~{::title},~{::content})">
+        <title>Your password has been updated</title>
+
+        <content>
+            <div class="container">
+                <div class="grid-row">
+                    <div class="column-two-thirds">
+
+                        <h1 class="heading-large">Your password has been updated</h1>
+                        <p>Please <a href="/login">login</a> to continue</p>
```

Comment:
FLOW - We should be sending the user to LPG

Reply:
We should always send the user to LPG UI regardless of where the came from? I thought this was supposed to be an oauth2 service?

Reply:
I know your point but just send to LPG makes life easier for us

Reply:
It already goes to LPG after login anyway because of this line in the config...

https://github.com/Civil-Service-Human-Resources/identity-service/blob/d36043751548d0db1682af24eb46fc838ee01831/src/main/java/uk/gov/cshr/config/UserSecurityConfig.java#L36

I can also hardcode it if that's what you want.